core/ledger/confighistory/mgr_test.go

```diff
@@ -257,6 +260,198 @@ func TestWithImplicitColls(t *testing.T) {
+			"chaincode1": cc1collConfigPackage,
+			"chaincode2": cc2collConfigPackage,
+			"chaincode3": cc3collConfigPackage,
+		},
+		100,
+	)
+	assert.NoError(t, err)
+	assert.NoError(t, dbHandle.writeBatch(batch, true))
+
+	fileHashes, err = env.retriever.ExportConfigHistory(env.testSnapshotDir, sha256.New())
+	require.NoError(t, err)
+
+	cc1configBytes, err = proto.Marshal(cc1collConfigPackage)
+	require.NoError(t, err)
+	cc2configBytes, err = proto.Marshal(cc2collConfigPackage)
+	require.NoError(t, err)
+	cc3configBytes, err = proto.Marshal(cc3collConfigPackage)
+	require.NoError(t, err)
+	expectedCollectionConfigs = append(
+		expectedCollectionConfigs,
+		&compositeKV{&compositeKey{ns: "lscc", key: "chaincode1~collection", blockNum: 100}, cc1configBytes},
+	)
+	expectedCollectionConfigs = append(
+		expectedCollectionConfigs,
+		&compositeKV{&compositeKey{ns: "lscc", key: "chaincode2~collection", blockNum: 100}, cc2configBytes},
+	)
+	expectedCollectionConfigs = append(
+		expectedCollectionConfigs,
+		&compositeKV{&compositeKey{ns: "lscc", key: "chaincode3~collection", blockNum: 100}, cc3configBytes},
+	)
+	verifyExportedConfigHistory(t, env.testSnapshotDir, fileHashes, expectedCollectionConfigs)
+}
+
+func verifyExportedConfigHistory(t *testing.T, dir string, fileHashes map[string][]byte, expectedCollectionConfigs []*compositeKV) {
+	require.Len(t, fileHashes, 2)
+	require.Contains(t, fileHashes, snapshotDataFileName)
+	require.Contains(t, fileHashes, snapshotMetadataFileName)
+
+	dataFile := path.Join(dir, snapshotDataFileName)
+	dataFileContent, err := ioutil.ReadFile(dataFile)
+	require.NoError(t, err)
+	dataFileHash := sha256.Sum256(dataFileContent)
+	require.Equal(t, dataFileHash[:], fileHashes[snapshotDataFileName])
+
+	metadataFile := path.Join(dir, snapshotMetadataFileName)
+	metadataFileContent, err := ioutil.ReadFile(metadataFile)
+	require.NoError(t, err)
+	metadataFileHash := sha256.Sum256(metadataFileContent)
+	require.Equal(t, metadataFileHash[:], fileHashes[snapshotMetadataFileName])
+
+	metadataReader, err := snapshot.OpenFile(metadataFile, snapshotFileFormat)
+	require.NoError(t, err)
+	defer metadataReader.Close()
+
+	dataReader, err := snapshot.OpenFile(dataFile, snapshotFileFormat)
+	require.NoError(t, err)
+	defer dataReader.Close()
+
+	namespace, err := metadataReader.DecodeString()
+	require.NoError(t, err)
+	require.Equal(t, collectionConfigNamespace, namespace)
+	numCollectionConfigs, err := metadataReader.DecodeUVarInt()
+	require.NoError(t, err)
+
+	var retrievedCollectionConfigs []*compositeKV
+	for i := uint64(0); i < numCollectionConfigs; i++ {
+		key, err := dataReader.DecodeBytes()
+		require.NoError(t, err)
+		val, err := dataReader.DecodeBytes()
+		require.NoError(t, err)
+		retrievedCollectionConfigs = append(retrievedCollectionConfigs,
+			&compositeKV{decodeCompositeKey(key), val},
+		)
+	}
+	require.ElementsMatch(t, expectedCollectionConfigs, retrievedCollectionConfigs)
```

Comment:
`require.Equal` as you would want this to be a deterministic order.

Reply:
Done.